scripts/eventbrite-API.py

```diff
@@ -67,8 +86,14 @@
                       'page': page
                       }
 
-            events_response = eventbrite.event_search(**params)
-            events = events_response['events']
+            events_response = Eventbrite(oauth_token).event_search(**params)
```

Comment:
Yeah, so I think we need to re-authenticate every time we swap tokens, or else the rate limit is triggered, since Eventbrite thinks we're still passing the old token.

Reply:
Yes, yes, re-authenticate every time you swap tokens but not every time you make a call to the API.